resources/home/dnanexus/get_variant_info.py

```
@@ -1,501 +1,481 @@
+    Work out inheritance of variant based on zygosity of parents
+    Inputs:
+        variant: (dict): dict extracted from JSON describing single variant
+        mother_idx: (int): position within list of variant calls that
+        belongs to the mother, or None if no mother in JSON
+        father_idx: (int): position within list of variant calls that
+        belongs to the father, or None if no father in JSON
+        p_sex (str): proband sex
+    Outputs:
+        inheritance (str): inferred inheritance of the variant, or None.
+    '''
+    zygosity = lambda x, y: x['variantCalls'][y]['zygosity']
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `zygosity` as a `def`

(E731)